charts/eth-validator-watcher/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 description: A Helm chart for running eth-validator-watcher
 name: eth-validator-watcher
 type: application
-version: 1.0.0
+version: 1.1.0
```

Comment:
We can include the appVersion field here to use it as an image tag instead of latest.